src/Discord.Net.WebSocket/Entities/Interaction/Message Components/SocketMessageComponent.cs

```diff
@@ -71,18 +68,21 @@ internal override void Update(Model model)
         }
 
         /// <inheritdoc/>
-        public override async Task RespondAsync(Embed[] embeds = null, string text = null, bool isTTS = false, InteractionResponseType type = InteractionResponseType.ChannelMessageWithSource,
-            bool ephemeral = false, AllowedMentions allowedMentions = null, RequestOptions options = null, MessageComponent component = null)
+        public override async Task RespondAsync(
+            string text = null,
+            Embed[] embeds = null,
+            bool isTTS = false,
+            bool ephemeral = false,
+            AllowedMentions allowedMentions = null,
+            RequestOptions options = null,
+            MessageComponent component = null)
         {
-            if (type == InteractionResponseType.Pong)
```

Comment:
Maybe keep this in? Why was it removed?

Reply:
It removed because i remove type argument